LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I am not sure if we can add this here. The data used in each workload comes with its own license and I think we may need to make sure we're not overriding the licenses here.

Each workload had its own license which resides at the end of each workload's README file. See example : https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/eventdata/README.md

Reply:
I remove this file and raise another PR